icon-font/css-template.css

```diff
@@ -3,29 +3,33 @@
  * This file is automatically built using a build process
  * If you need to fix errors, see https://github.com/WordPress/dashicons
  */
+
+/* stylelint-disable function-url-quotes, declaration-colon-newline-after */
 <% if (fontfaceStyles) { %>
 <% if (fontSrc1 && embed.length) { %>@font-face {
-	font-family: "<%= fontFamilyName %>";
+	font-family: <%= fontFamilyName %>;
 	src: <%= fontSrc1 %>;
-	font-weight: normal;
+	font-weight: 400;
```

Comment:
I'm guessing this is some polish changes while you were at it, i.e. it should be 400 and not just "normal"?

Reply:
Yes, via https://make.wordpress.org/core/handbook/best-practices/coding-standards/css/#values:
> _Font weights should be defined using numeric values (e.g. `400` instead of `normal`, `700` rather than `bold`)._